sdk/cosmos/azure-cosmos/azure/cosmos/_auth_policy.py

```diff
@@ -5,33 +5,29 @@
 # -------------------------------------------------------------------------
 import time
 
-from typing import Any, Dict, Optional
-from azure.core.credentials import AccessToken
+from typing import Any, MutableMapping, Optional
+from azure.core.credentials import AccessToken, TokenCredential
 from azure.core.pipeline import PipelineRequest, PipelineResponse
 from azure.core.pipeline.policies import HTTPPolicy
 from azure.cosmos import http_constants
 
 
 # pylint:disable=too-few-public-methods
-class _CosmosBearerTokenCredentialPolicyBase(object):
+class _CosmosBearerTokenCredentialPolicyBase:
     """Base class for a Bearer Token Credential Policy.
 
     :param credential: The credential.
     :type credential: ~azure.core.credentials.TokenCredential
     :param str scopes: Lets you specify the type of access needed.
     """
 
-    def __init__(self, credential, *scopes, **kwargs):  # pylint:disable=unused-argument
-        # type: (TokenCredential, *str, **Any) -> None
-        super(_CosmosBearerTokenCredentialPolicyBase, self).__init__()
+    def __init__(self, credential: TokenCredential, *scopes: str) -> None:  # pylint:disable=unused-argument
```

Comment:
Not sure why we had `kwargs` here - I will did deep to confirm whether removing it is appropriate.